src/test/java/com/karankumar/bookproject/backend/service/AuthorServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.karankumar.bookproject.backend.service;
+
+import com.karankumar.bookproject.annotations.IntegrationTest;
+import com.karankumar.bookproject.backend.entity.Author;
+import com.karankumar.bookproject.backend.repository.AuthorRepository;
+import lombok.extern.java.Log;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Log
+@IntegrationTest
+public class AuthorServiceTest {
+
+    private final AuthorService sut;
```

Comment:
By `sut`, I know you mean 'system under test', but this may not be clear to others. Hence, could you please rename this to the more readable `authorService`?